### PR TITLE
Alpine 3.8 for ARM64 

### PR DESCRIPTION
**Pull request by ulisesh**

### CODE REVIEW

**Comment by ulisesh** [comment]
@MichaelSimons, can you please queue a test build for this PR?

File: src/alpine/3.8/helix/arm64v8/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        python-dev \
+        py-pip \
+        sudo \
+        tzdata \
+        util-linux-dev \
+        wget \
+        which \        
+        zlib-dev && \
+    apk -X http://dl-cdn.alpinelinux.org/alpine/edge/main add --no-cache \
+        userspace-rcu-dev \
```
Comment (MichaelSimons) [inline src/alpine/3.8/helix/arm64v8/Dockerfile]: (nit) - alphabetize.

Comment (ulisesh) [reply src/alpine/3.8/helix/arm64v8/Dockerfile]: I thought I had the order correct but I missed a couple

File: src/alpine/3.8/helix/arm64v8/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        py-pip \        
+        sudo \
+        tzdata \
+        util-linux-dev \
+        wget \
+        which \        
+        zlib-dev && \
+    apk -X http://dl-cdn.alpinelinux.org/alpine/edge/main add --no-cache \
+        userspace-rcu-dev \
+        lttng-ust-dev
```
Comment (MichaelSimons) [inline src/alpine/3.8/helix/arm64v8/Dockerfile]: Looks like `lttng-ust-dev` should be moved up one line.

**Comment by MichaelSimons** [comment]
The following image(s) were produced as a result of merging this PR:

mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.8-helix-arm64v8-46e69dd-20190313223330

**Pull request merged**